src/sources/forus-webshop/js/angular-1/i18n/nl/pages/signup.js

```diff
@@ -20,6 +21,19 @@ module.exports = {
         vergoedingen: {
             title: "Inloggen",
             restore_title: "Account herstellen",
+            signup_option: "Log in met DigiD om verder te gaan",
+        },
+        groningen: {
```

Comment:
should we put it in qdt-env later as a low prio refactoring? 
this structure repeats qdt-env structure ( list of implementations ), so now we will have one more place with implementations lists in frontend

Reply:
discussed - postponed to be in react webshop issue
https://github.com/teamforus/AMR-2022-Angular-migration-react/issues/23